app/models/user.rb

```diff
@@ -1,20 +1,81 @@
 class User < ApplicationRecord
-  has_many :issues, dependent: :destroy
+  has_many :issues, dependent: :destroy # created issues
+  has_and_belongs_to_many :social_interests, class_name: "SocialInterest"
```

Comment:
I thought of giving another name for member var name, that's why put a class there. For my reference, I keep it for now.